packages/eslint-config-next/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "eslint-config-next",
+  "version": "10.1.3",
+  "description": "ESLint configuration used by NextJS.",
+  "main": "index.js",
+  "license": "MIT",
+  "repository": {
+    "url": "vercel/next.js",
+    "directory": "packages/eslint-config-next"
+  },
+  "dependencies": {
+    "@rushstack/eslint-patch": "^1.0.6",
+    "@babel/core": "^7.13.14",
+    "@babel/eslint-parser": "^7.13.14",
+    "@next/eslint-plugin-next": "^10.1.3",
+    "@typescript-eslint/parser": "^4.20.0",
+    "eslint": "^7.23.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-react": "^7.23.1",
+    "eslint-plugin-react-hooks": "^4.2.0"
+  },
+  "peerDependencies": {
+    "eslint": "^7.23.0"
+  }
```

```suggestion
    "eslint": "^7.23.0",
    "typescript": ">=3.3.1",
   },
  "peerDependenciesMeta": {
    "typescript": {
      "optional": true
    }
  }
```
Needed for `@typescript-eslint/parser`